packages/data-broker/src/tests/services/weather/WeatherService.test.js

```diff
@@ -136,10 +136,30 @@ describe('WeatherService', () => {
         },
       ]);
     });
+
+    it('throws when no dates are provided', async () => {
+      const mockModels = await createMockModelsStubWithMockEntity();
+
+      const mockApi = createWeatherApiStubWithMockResponse();
+
+      const service = new WeatherService(mockModels, mockApi);
+
+      const functionCall = async () =>
```

Comment:
```suggestion
      const pullWithUndefinedDates = async () =>
```
With `functionCall` I have to think a bit harder about "ok, but what function call?"